lib/svtplay_dl/utils/retriever_multi.py

```diff
@@ -0,0 +1,113 @@
+# ex:ts=4:sw=4:sts=4:et
+# -*- tab-width: 4; c-basic-offset: 4; indent-tabs-mode: nil -*-
+
+import pycurl
+from io import BytesIO
+import time
+from datetime import timedelta
+
+# We should ignore SIGPIPE when using pycurl.NOSIGNAL - see
+# the libcurl tutorial for more info.
+try:
+    import signal
+    from signal import SIGPIPE, SIG_IGN
+except ImportError:
+    pass
+else:
+    signal.signal(SIGPIPE, SIG_IGN)
+
+from svtplay_dl.utils.output import progressbar, ETA
+from svtplay_dl.utils import http
+
+DEBUG = False
+
+
+def get(urls, cookies, config=None, silent=False, live=False, total_duration=None, tot_urls=0, num_conn=100):
```

Comment:
`num_conn` should be a config value.   maybe a lower value as default.

Reply:
Now I use `os.cpu_count()`

Reply:
Ex:
```
  -j jobs, --jobs jobs  Allow N download jobs at once. Default automatic
                        thread usage.

```